ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/CatalogController.java

```diff
@@ -503,13 +501,8 @@ public void catalogDeleteCheck(Context ctx) {
   public void deleteCatalog(Context ctx) {
     String catalogName = ctx.pathParam("catalogName");
     Preconditions.checkArgument(StringUtils.isNotEmpty(ctx.pathParam("catalogName")), "Catalog name is empty!");
-    List<String> dbs = tableService.listDatabases(catalogName);
```

Comment:
We should check whether the catalog is an external catalog, and only the external catalog can skip the checking before dropping it from AMS.